provider/resources.go

```diff
@@ -124,6 +124,32 @@ func Provider() tfbridge.ProviderInfo {
 			"artifactory_release_bundle_v2_promotion": {ComputeID: computeIDField("name")},
 			"artifactory_item_properties":             {ComputeID: computeIDField("repoKey")},
 
+			// ComputeID mappings for v12.2.0 on webhooks
+			"artifactory_artifact_custom_webhook":                    {ComputeID: computeIDField("id")},
```

Comment:
Looking at [upstream's documentation](https://registry.terraform.io/providers/jfrog/artifactory/latest/docs/resources/artifact_custom_webhook), I don't see an `"id"` attribute to reference here, which means this will panic at runtime.
```suggestion
			"artifactory_artifact_custom_webhook":                    {ComputeID: computeIDField("key")},
```

I would use the `"key"` attribute here.